clients/metadata/metadata_cassandra.go

```diff
@@ -3703,6 +3798,81 @@ func (s *CassandraMetadataService) readConsumerGroupExtentsHelper(request *m.Rea
 	return query.Iter(), nil
 }
 
+// ReadConsumerGroupExtentsLite returns the list all extents mapped to
+// the given consumer group. This API only returns a few interesting
+// columns for each extent in the result. For detailed info about
+// extents, see ReadConsumerGroupExtents
+func (s *CassandraMetadataService) ReadConsumerGroupExtentsLite(ctx thrift.Context, request *m.ReadConsumerGroupExtentsLiteRequest) (*m.ReadConsumerGroupExtentsLiteResult_, error) {
+
+	if request.GetMaxResults() < 1 {
```

Comment:
nit: in 'ListDestinationExtents', your corresponding check is "<=0"; might want to make it consistent .. ;-)